helpers.php

```diff
@@ -16,3 +16,45 @@ function retry($retries, $fn)
         goto beginning;
     }
 }
+
+if (! function_exists('abort_if')) {
```

Comment:
Yeah, I only figured out I had to add these after `abort_if` failed running the unit tests 😄